### ISSUE TITLE
Drawing on a Motion Tween frame causes shapes to unexpectedly move

### ISSUE THREAD

**Issue by octoConnors** [zrispo/wick#681]
When I tried to draw on the already motion-tweened frame shapes were automatically added to the Group AND they were moved to be centered on it. The workaround is editing the group directly, so I would consider this a minor bug. There ought to be a failsafe, however; Either the program correctly orients shapes to the group when you draw them, or it should stop the user from drawing on a tweened frame by making a popup window with a warning, such as: "You shouldn't draw directly on a motion tween, but you can edit the Group itself if you want to draw on it!"

**Comment by zrispo** [comment]
In Flash if you try to add shapes to a frame that's already tweened, it wouldn't let you and give you a message saying you can't do that (basically the same as the popup functionality you described actually)

I'm wondering if we can avoid having a popup if any shapes drawn/added to the frame were automatically added to the group, but that might be confusing. Thoughts?

**Comment by octoConnors** [comment]
I agree that automatically adding shapes might be confusing. The expected behavior is that drawing on a frame creates new shapes, which doesn't work on a tween. The popup could help new users understand how tweens work by forcing them to click the grouped shapes and edit the group.

**Comment by kryptot7** [comment]
Why not rename the Tween Point / Motion Tween thing to "Layer Tween"? Then it will be clear that all movement, scaling, rotation, and/or color warping (when that's implemented) applies to the entire layer. It will subtly cue the user that "okay, I need to put all my tweening shapes on separate layers."

**Comment by Luxapodular** [comment]
Reviving this thread since tween points seem to be a recurring issue still. We may want to look into a rename like Kryptot suggests.
